### PR TITLE
feat:[24] Update Home page buttons

### PR DESCRIPTION
**Pull request by mulf0**

### CODE REVIEW

**Comment by mhmulford0** [comment]
closes #19 with Andrii

**Comment by mulf0** [comment]
closes #19

**Comment by kempsterrrr** [comment]
Looks great to me! Smashed it as always @andriishupta 🤝 

Noticed the figman design doesn't have the white box around the component - thoughts on having these white lines or not? cc @mhmulford0 - I'd vote for no white lines but not a massive thing

Screenshot 2022-08-21 at 22 06 57

File: packages/frontend/src/Components/Intro/index.tsx [hunk 1]
```diff
@@ -31,109 +27,34 @@ const Intro: FC<IntroProps> = ({ heading, subHeading, currentStatus }) => {
...
-          minW={{ base: '0', xl: '35rem' }}
-          margin="0.85rem"
-        >
+      <Box
+        w={{ base: '100%', xl: '40%' }}
+        minW={{ base: '0', xl: '34rem' }}
+        maxW={{ xl: '34rem' }}
+        margin={{ lg: '0.75rem' }}
+        border="1px solid"
+        borderRadius="11px"
```
Comment (kempsterrrr) [inline packages/frontend/src/Components/Intro/index.tsx]: No Border on the figma design here btw

**Comment by mulf0** [comment]
> Looks great to me! Smashed it as always @andriishupta 🤝 
> 
> 
> 
> Noticed the figman design doesn't have the white box around the component - thoughts on having these white lines or not? cc @mhmulford0 - I'd vote for no white lines but not a massive thing
> 
> 
> 
> Screenshot 2022-08-21 at 22 06 57
> 
> 
> 
> 

I'd say follow the design since it is the source of truth in this scenario.

**Comment by mulf0** [comment]
@kempsterrrr I think this resolves any outstanding items.

**Pull request merged**